syntax = "proto3";

package api.api_grpc;

message MessageRequest {
  string senderName = 1;
  string text = 2;
}

message MessageResponse {
  string result = 1;
}

service ApiService {
  rpc SendMessage(MessageRequest) returns (MessageResponse);
}